pkgs/games/warzone2100/default.nix

```diff
@@ -1,25 +1,25 @@
 { stdenv, fetchurl, bison, flex, gettext, pkgconfig, libpng
 , libtheora, openal, physfs, mesa, fribidi, fontconfig
 , freetype, qt4, glew, libogg, libvorbis, zlib, libX11
-, libXrandr, zip, unzip, which
+, libXrandr, zip, unzip, which, perl
 , withVideos ? false
 }:
 stdenv.mkDerivation rec {
   pname = "warzone2100";
-  version = "3.1.1";
+  version = "3.1.5";
   name = "${pname}-${version}";
   src = fetchurl {
     url = "mirror://sourceforge/${pname}/releases/${version}/${name}.tar.xz";
-    sha256 = "c937a2e2c7afdad00b00767636234bbec4d8b18efb008073445439d32edb76cf";
+    sha256 = "0hm49i2knvvg3wlnryv7h4m84s3qa7jfyym5yy6365sx8wzcrai1";
   };
-  sequences_src = fetchurl {
+  sequences_src = stdenv.lib.optionalString withVideos (fetchurl {
```

Comment:
That's right -- to be more specific, `b` would contain the string `"$b"`. `postInstall` and other hooks are just strings too, that get `eval`ed by Bash somewhere inside the generic build script. So if you would mention `$b` in `postInstall` Bash would attempt to substitute it by environment variable `b` whatever that might be (pun not intended ^^). On the other hand, `c` would contain the string `"ccc"`.